### PR TITLE
Update audio-config.md

### PR DESCRIPTION
**Pull request by Sprunglicious**
I added a section with instructions to get a sound profile similar to Dolby Atmos. I provided step-by-step instructions for those not very familiar with EasyEffects. This section was added into the EasyEffects and Pipewire section.

### CODE REVIEW

**Comment by AdityaGarg8** [comment]
Have you tested this yourself? Also address the lint issues

**Comment by Sprunglicious** [comment]
I have been using this configuration on my Macbook Pro 2018 (on Ubuntu)

**Review by Sprunglicious — COMMENTED** [review]
I fixed the typo and space.

File: docs/guides/audio-config.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ does not offer a great deal of flexibility, it might work for getting acceptable
 to the KDE approach mentioned above using input plugins like "Autogain" offers a lot more fine grain control and higher volume
 boosts.
 
+There is a preconfigured convolver file to give a Dolby Atmos-type sound profile. You can find it [here](https://github.com/JackHack96/EasyEffects-Presets/blob/master/irs/Dolby%20ATMOS%20((128K%20MP3))%201.Default.irs). You simply need to download this file, open EasyEffects, select "Effects" --> Add Effect --> type "Convolver". Afterward select "Impulses" in the rightside menu, select the "Import Impulses" option and then select the file you downloaded earlier. It will usually be in your ~/Downloads folder. As a bonus, go into the preferences of EasyEffects and make sure "Shutdown on window closing" is deselected and "Launch Service at System Startup" is selected. 
+
```
Comment (AdityaGarg8) [inline docs/guides/audio-config.md:61]: You need to remove the space from here

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: docs/guides/audio-config.md (hunk 1 above)
Comment (Sprunglicious) [reply docs/guides/audio-config.md:61]: Are you talking about the bottom line?

Comment (AdityaGarg8) [reply docs/guides/audio-config.md:61]: I meant space at line 60. Nvm, I've removed it.

**Comment by sharpenedblade** [comment]
I dont think we need this, its confusing and it subjective whether it actually sounds better.

**Pull request closed without merging**